ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/public-to-private/provision/cucushift-installer-rehearse-azure-ipi-public-to-private-provision-chain.yaml

```diff
@@ -8,8 +8,8 @@ chain:
   - chain: azure-provision-bastionhost
   - ref: proxy-config-generate
   - ref: azure-cluster-public-to-private
-  - ref: cucushift-installer-check-azure-internal-registry
   - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-azure-internal-registry
```

Comment:
@jinyunma Is this step moved to the back to wait for image-registry to be fully updated? If so, do we need to further enhance the judgment here? https://steps.ci.openshift.org/reference/azure-cluster-public-to-private#line139

Reply:
hmm, I don't plan to move this step back. From [the result of step cucushift-installer-check-operators](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/68875/rehearse-68875-periodic-ci-openshift-openshift-tests-private-release-4.20-multi-nightly-azure-ipi-public-to-private-arm-f28/1963530602841903104/artifacts/azure-ipi-public-to-private-arm-f28/cucushift-installer-check-operators/build-log.txt), co image-registry became updating again after checks in [azure-cluster-public-to-private#line139](https://steps.ci.openshift.org/reference/azure-cluster-public-to-private#line139) got passed.

So let step cucushift-installer-check-operators do operator check, then do azure-internal-registry post check.
wdyt?